apps/web/public/sw.js

```
@@ -87,6 +87,32 @@ self.addEventListener("push", (event) => {
+    }
+  }
+  if (event.data?.type === "SKIP_WAITING") {
+    self.skipWaiting()
+  }
+})
+
+// Re-subscribe if the browser rotates push keys
+self.addEventListener("pushsubscriptionchange", (event) => {
+  event.waitUntil(
+    self.clients.matchAll({ type: "window" }).then((clients) => {
+      clients.forEach((client) => client.postMessage({ type: "PUSH_SUBSCRIPTION_CHANGED" }))
```

Biome (2.4.6)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)